1.3/Patches/z_add_chitchat_thoughts.xml

```diff
@@ -96,6 +110,8 @@
 							<li>extra->and I don't like it.</li>
 							<li>extra->I hope I won't trip into something.</li>
 							<li>extra->I can't see well enough to do my job properly.</li>
+							<li>extra->I can't see anything.</li>
+							<li>extra->and that's mysterious.</li>
```

Comment:
Mysterious? I did not understand :-) For dark places in general?

Reply:
Yes - dark areas are full of mystery :)
This leads to pawns saying "It's dark - and that's mysterious." when they're wandering around in the dark.